### PR TITLE
Update variables.tf

### PR DESCRIPTION
**Pull request by joealewine**
### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by SirSpidey — APPROVED** [review]
Made some updates for consistency with other DAs.

**Review by MatthewLemmond — APPROVED** [review]

File: solutions/standard/variables.tf [hunk 1]
```diff
@@ -16,13 +16,13 @@ variable "use_existing_resource_group" {
...
   type        = string
-  description = "The name of a new or an existing resource group in which to provision KMS resources to. If prefix input variable is passed then it will get prefixed infront of the value in the format of '<prefix>-value'."
+  description = "The name of a new or an existing resource group in which to provision key management resources to. If a prefix input variable is specified, it's added to the value in the `<prefix>-value` format."
 }
 
 variable "region" {
   type        = string
   default     = "us-south"
-  description = "The region in which to provision KMS resources. If using existing KMS, set this to the region in which it is provisioned in."
+  description = "The region in which to provision key management resources. If using an existing Key Protect or Hyper Protect Crypto Services key, set this to the region where it was provisioned."
```
Comment (MatthewLemmond) [inline solutions/standard/variables.tf:25]: should be `If using an existing Key Protect or Hyper Protect Crypto Services instance`

File: solutions/standard/variables.tf [hunk 2]
```diff
@@ -38,12 +38,12 @@ variable "prefix" {
 variable "key_protect_instance_name" {
   type        = string
   default     = "base-security-services-kms"
-  description = "The name to give the Key Protect instance that will be provisioned by this solution. Only used if not supplying an existing KMS instance. If prefix input variable is passed then it will get prefixed infront of the value in the format of '<prefix>-value'."
+  description = "The name to give the Key Protect instance that will be provisioned by this solution. Only used if not supplying an existing Key Protect instance. If a prefix input variable is specified, it's added to the value in the `<prefix>-value` format."
```
Comment (MatthewLemmond) [inline solutions/standard/variables.tf:41]: `Only used if not supplying an existing Key Protect or Hyper Protect Crypto Services instance.`

File: solutions/standard/variables.tf [hunk 3]
```diff
@@ -38,12 +38,12 @@ variable "prefix" {
...
 }
 
 variable "key_protect_allowed_network" {
   type        = string
-  description = "The type of the allowed network to be set for the Key Protect instance. Possible values are 'private-only', or 'public-and-private'. Only used if not supplying an existing KMS instance."
+  description = "The type of the allowed network to be set for the Key Protect instance. Possible values: `private-only`, `public-and-private`. Applies only if an existing Key Protect instance is not specified."
```
Comment (MatthewLemmond) [inline solutions/standard/variables.tf:46]: `Applies only if an existing Key Protect or Hyper Protect Crypto Services instance`

File: solutions/standard/variables.tf [hunk 4]
```diff
@@ -53,19 +53,19 @@ variable "key_protect_allowed_network" {
 
 variable "resource_tags" {
   type        = list(string)
-  description = "Optional list of tags to be added to the Key Protect instance. Only used if not supplying an existing KMS instance."
+  description = "Optional list of tags to be added to the Key Protect instance. Only used if not supplying an existing Key Protect instance."
```
Comment (MatthewLemmond) [inline solutions/standard/variables.tf:56]: `Key Protect or Hyper Protect Crypto Services instance.`

File: solutions/standard/variables.tf [hunk 5]
```diff
@@ -53,19 +53,19 @@ variable "key_protect_allowed_network" {
...
   default     = []
 }
 
 variable "access_tags" {
   type        = list(string)
-  description = "Optional list of access tags to apply to the Key Protect instance. Only used if not supplying an existing KMS instance."
+  description = "Optional list of access tags to apply to the Key Protect instance. Only used if not supplying an existing Key Protect instance."
```
Comment (MatthewLemmond) [inline solutions/standard/variables.tf:62]: `Key Protect or Hyper Protect Crypto Services instance.`

File: solutions/standard/variables.tf [hunk 6]
```diff
@@ -53,19 +53,19 @@ variable "key_protect_allowed_network" {
...
   default     = []
 }
 
 variable "rotation_interval_month" {
   type        = number
-  description = "Specifies the key rotation time interval in months. Must be between 1 and 12 inclusive. Only used if not supplying an existing KMS instance."
+  description = "Specifies the key rotation time interval in months. Possible values: `1` through `12`. Applies only if an existing Key Protect instance is not specified."
```
Comment (MatthewLemmond) [inline solutions/standard/variables.tf:68]: `Key Protect or Hyper Protect Crypto Services instance.`

**Review by MatthewLemmond — CHANGES_REQUESTED** [review]

File: solutions/standard/variables.tf (hunk 4 above)
Comment (joealewine) [reply solutions/standard/variables.tf:56]: The variable is literally `key_protect_allowed_network` -- how can HPCS be relevant here?

**Review by MatthewLemmond — APPROVED** [review]

**Pull request merged**